src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I believe the middle line here needs to come out if it's Mono code:
```
	1. Keep the existing copyright headers in place
		○ Note: If you are porting just portions of the file over, you still need to bring over all the existing copyright headers.
	2. Add the following copyright headers – NOTE this is missing the middle line of the ‘regular’ CoreFx copyright headers:
 // Licensed to the .NET Foundation under one or more agreements.
// See the LICENSE file in the project root for more information.
```
Although if only part is, I'm not sure. @AlexGhiondea did legal indicate what to do in that case?